src/Components/Scenes/RequestAssets/RequestedBy/RequestedBy-Container.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+import RequestedBy from './RequestedBy';
+
+// This will get updated when Auth is done
+const mapStateToProps = ({ requestAssets }) => {
+  return {
+    requestedBy: `Scott Fowles`
```

Comment:
FYI:  I don't see a human friendly name on the SAML token.  We may need to have an additional call that gets the currentUsers details.

Maybe there could be a new endpoint that that returns user details from an email address.  Perhaps this call could become the userPreferences down the road.

Reply:
We could alternatively display email here, if there's no good way to deduce the friendly name. Worth considering that we have lots of folks here with hyphenated first and/or last names

Reply:
@davidkliewer-kirby  It seems to me more of userPreference. I think SAML token provide employee email.
@sfowles  thats good point. I just checked the old Kirby we are using email there. So i will display email.